gmso/core/angle.py

```diff
@@ -31,6 +31,16 @@ class Angle(Connection):
         default=None, description="AngleType of this angle."
     )
 
+    restraint_: Optional[dict] = Field(
+        default=None,
+        description="""
+        Restraint for this angle, must be a dict with the following keys:
```

Comment:
Do we plan to do something different to support ["constraints"](https://docs.lammps.org/fix_restrain.html) as opposed to ["restraints"](https://docs.lammps.org/fix_rigid.html)? It looks like we'll need a different implementation for that according to LAMMPS, which has both.

Reply:
yes, I can see how constraint and restraint co-exist, may be in a PR improving the LAMMPS writer